packages/showcases/css/stories/components/navigation/tabs/examples/demo.html

```diff
@@ -1,21 +1,196 @@
 <div class="block">
-  <nav id="demo-tabs-nav" aria-label="Tabs list" class="vtmn-tabs">
-    <ul id="demo-tabs-tablist" role="tablist">
-      <span id="block-focus-trap-start" tabindex="-1"></span>
-      <li id="tab-demo-1" role="tab" aria-selected="false">
-        <a href="" class="selected">Tab Name</a>
+  <nav id="demo-tabs-nav" class="vtmn-tabs">
+    <ul id="demo-tabs-tablist" role="tablist" aria-label="Tabs navigation">
+      <li>
+        <a
+          href="#tabpanel-demo-1"
+          role="tab"
+          id="tab-demo-1"
+          aria-controls="tabpanel-demo-1"
+          >Tab Name 1</a
+        >
       </li>
-      <li id="tab-demo-2" role="tab" aria-selected="false">
-        <a href="">Tab Name</a>
+      <li>
+        <a
+          href="#tabpanel-demo-2"
+          role="tab"
+          id="tab-demo-2"
+          aria-controls="tabpanel-demo-2"
+          >Tab Name 2
+        </a>
       </li>
-      <li id="tab-demo-3" role="tab" aria-selected="false">
-        <a href="">Tab Name</a>
+      <li>
+        <a
+          href="#tabpanel-demo-3"
+          role="tab"
+          id="tab-demo-3"
+          aria-controls="tabpanel-demo-3"
+          >Tab Name 3</a
+        >
       </li>
-      <li id="tab-demo-4" role="tab" aria-selected="false">
-        <a href="">Tab Name</a>
+      <li>
+        <a
+          href="#tabpanel-demo-4"
+          role="tab"
+          id="tab-demo-4"
+          aria-controls="tabpanel-demo-4"
+          >Tab Name 4</a
+        >
       </li>
-      <span id="block-focus-trap-end" tabindex="-1"></span>
     </ul>
     <div class="vtmn-divider" role="separator"></div>
   </nav>
+  <div
+    id="demo-tabs-tabpanels"
+    class="vtmn-w-full vtmn-p-5 vtmn-bg-background-primary"
+  >
+    <section
+      role="tabpanel"
+      id="tabpanel-demo-1"
+      aria-labelledby="tab-demo-1"
+      tabindex="0"
+    >
+      Tab Name 1 <br />
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Velit tenetur
+      voluptatum asperiores quis eum perspiciatis repellat temporibus ab aperiam
+      cumque deserunt eveniet dolore excepturi, error veniam quidem ipsum
+      necessitatibus? Sequi?
+    </section>
```

Comment:
@GaspardMathon The text is a single node, what's the reason to put it on multiple lines? It makes it harder to manage no?

Reply:
I think the prettier config automatically set long texts on several lines. @thibault-mahe